composite/proof-of-reserves/src/balance.ts

```diff
@@ -9,9 +9,7 @@ import btcCom from '@chainlink/btc.com-adapter'
 import cryptoapis from '@chainlink/cryptoapis-adapter'
 import sochain from '@chainlink/sochain-adapter'
 
-const ENV_BTC_INDEXER_ADAPTER = 'BTC_INDEXER_ADAPTER'
-
-const adapters: AdapterImplementation[] = [
+export const adapters: AdapterImplementation[] = [
```

Comment:
I kept the dependencies on the underlying adapters purely to use them as a source of truth for `.NAME`, though I could see removing this. Thoughts?